mattermost_giphy/settings.py

```diff
@@ -16,5 +16,6 @@
 # the is a public beta key from giphy api
 GIPHY_API_KEY = os.environ.get('GIPHY_API_KEY', 'dc6zaTOxFJmzC')
 
-# the Mattemost token generated when you created your outgoing webhook
+# the Mattermost token or tokens generated when you created your outgoing webhook
+# multiple tokens needs to be separated by a colon
 MATTERMOST_GIPHY_TOKEN = os.environ.get('MATTERMOST_GIPHY_TOKEN', None)
```

Comment:
In my opinion, it would be better to split one time here ? something like :
`MATTERMOST_GIPHY_TOKEN = os.environ.get('MATTERMOST_GIPHY_TOKEN', '').split(':')`